.github/workflows/common-tests.jl

```diff
@@ -0,0 +1,11 @@
+if !occursin("hostedtoolcache", Sys.BINDIR)
```

Comment:
I'd consider moving this into a `.github/bin` or `.github/scripts` directory as this file isn't a GitHub workflow